backend/coins/eth/account.go

```diff
@@ -215,7 +216,25 @@ func (account *Account) Initialize() error {
 }
 
 func (account *Account) poll(initDone func()) {
-	timer := time.After(0)
+	updateAccount := func() {
+		balance, err := account.coin.client.Balance(context.TODO(), account.address.Address)
```

Comment:
Why is there a poll loop here and in backend.go? Isn't the account now being updated twice every 5mins (pollInterval)?

Reply:
This is not a real loop anymore, as it doesn't use the timer; it just waits for the enqueueUpdateChan to perform the update of a single account.I can probably see if it can be removed and merged into the backend loop though

Reply:
Ah I missed that it has no timer anymore. But as per the other comment, since the backend poll loop will also need a way to quit and enqueue, I guess we can do away with this function altogether.

Be mindful of `initDone()`, which takes care of the ETH account showing a loading screen until the first results are available. I guess this is a very clumsy way of achieving that (inherited from the BTC account where there are a lot more calls to sync), maybe one day we can do away with the sync counter.